packages/calypso-color-schemes/src/shared/color-schemes/_coffee.scss

```diff
@@ -0,0 +1,135 @@
+/*
+WP Admin Unthemed definitions:
+
+$text-color: #fff !default;
+$base-color: #23282d !default;
+$icon-color: hsl( hue( $base-color ), 7%, 95% ) !default;
+$highlight-color: #0073aa !default;
+$notification-color: #d54e21 !default;
+
+WP Admin Coffee Definition:
+
+$base-color: #59524c;
+$highlight-color: #c7a589;
+$notification-color: #9ea476;
+$low-contrast-theme: "true";
+
+$form-checked: $base-color;
+
+Used studio-orange for the primary+accent.
+The wp-admin highlight color hue is 27, while studio-orange ranges from 25 to 35.
+*/
+
+// Set up variables for Calypso ectoplasm
+:root {
+    --theme-text-color: #ffffff; /* Direct from wp-admin */
+    --theme-text-color-rgb: 255, 255, 255; /* Manual conversion */
+    --theme-base-color: #59524c; /* Direct from wp-admin */
+    --theme-base-color-rgb: 89, 82, 76; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-submenu-background-color: #46403c; /* darken( $base-color, 7% ), computed: http://scg.ar-ch.org/
+    --theme-icon-color: #ece6f6; /* Direct from wp-admin */
+    --theme-highlight-color: #c7a589; /* Direct from wp-admin */
+    --theme-highlight-color-rgb: 199, 165, 137; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-notification-color: #9ea476; /* Direct from wp-admin */
+}
```

Comment:
As also suggested in https://github.com/Automattic/wp-calypso/pull/47122 let's try consolidating these variables in _defaut.scss.

Reply:
Done